generate/create_detection_data.py

```diff
@@ -88,45 +91,48 @@ def create_detector_data(dataset_name, dataset_path, image_name, image, data):
 
 def convert_data(dataset_type, num, offset=0):
 
-    new_dataset = 'detector_' + dataset_type
+    new_dataset = ('detector_' + dataset_type, ) * num
     images_path = os.path.join(config.DATA_DIR, dataset_type, 'images')
-    new_images_path = os.path.join(config.DATA_DIR, new_dataset, 'images')
+    new_images_path = (os.path.join(config.DATA_DIR, new_dataset[0], 'images'), ) * num
```

Comment:
Will be an error if no train or validation images generated.

Reply:
Fixed this by adding an if statement and breaking if num is 0